src/PluginSuppression.php

```diff
@@ -212,6 +212,7 @@ public function sanitize_options( $options, $new_options ) {
 				// Remove the plugin from being suppressed.
 				unset( $option[ $plugin_slug ] );
 			} elseif ( ! isset( $option[ $plugin_slug ] ) && $suppressed && array_key_exists( $plugin_slug, $plugins ) ) {
+				/** @var WP_User|null */
```

Comment:
I guess `wp_get_current_user()` will _always_ return a `WP_User` instance. In the case of an unauthenticated user, the user object will have an `ID` of `0`. In this case, it would be better to remove this and change the below as follows:

```diff
- Option::SUPPRESSED_PLUGINS_USERNAME  => $user instanceof WP_User ? $user->user_nicename : null,
+ Option::SUPPRESSED_PLUGINS_USERNAME  => $user->id ? $user->user_nicename : null,
```